packages/airnode-examples/test/example-files-are-up-to-date.test.ts

```diff
@@ -18,7 +18,10 @@ describe('Verifies that all config.example.json files are up to date', () => {
 
       if (!goCreateConfig.success) throw goCreateConfig.error;
 
-      const generatedConfigFile = readFileSync(path).toString();
+      const generatedConfigFile = readFileSync(path)
+        .toString()
+        // fix prettier newline disagreement with generated config for `coingecko-e2e`
```

Comment:
I don't think you will need this.

Reply:
Unfortunately, this is needed.

Reply:
Mhh, can you explain a bit more what is the issue?

Reply:
Without this (hack) you get the following test failure (notice the newline disagreement) when running this test for this new `coingecko-e2e` integration:

```ts
Verifies that all config.example.json files are up to date

    expect(received).toBe(expected) // Object.is equality

    - Expected  - 2
    + Received  + 6

    @@ -1,14 +1,18 @@
      {
        "chains": [
          {
            "maxConcurrency": 100,
            "authorizers": {
    -         "requesterEndpointAuthorizers": ["0xE2E0000000000000000000000000000000000000"],
    +         "requesterEndpointAuthorizers": [
    +           "0xE2E0000000000000000000000000000000000000"
    +         ],
              "crossChainRequesterAuthorizers": [
                {
    -             "requesterEndpointAuthorizers": ["0xE2E1111111111111111111111111111111111111"],
    +             "requesterEndpointAuthorizers": [
    +               "0xE2E1111111111111111111111111111111111111"
    +             ],
                  "chainType": "evm",
                  "chainId": "31337",
                  "contracts": {
                    "AirnodeRrp": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
                  },

      26 |       writeFileSync(path, currentConfigFile);
      27 |
    > 28 |       expect(generatedConfigFile).toBe(currentConfigFile);
         |                                   ^
      29 |     });
      30 |   });
      31 | });

      at test/example-files-are-up-to-date.test.ts:28:35
      at fulfilled (test/example-files-are-up-to-date.test.ts:28:58)
```

Reply:
Ahh, I see. Can you try fixing that by formatting the generated file? See: https://prettier.io/docs/en/api.html

Reply:
Found the issue- the generated config didn't obey our prettier `printWidth` of 120. Your suggestion worked, thanks 👍